.github/workflows/create-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Create GitHub release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own
+        with:
+          tag_name: ${{ github.ref }}
```

Comment:
Since the trigger won't be a new tag, I'll need to figure out how to parse the tag name here and below

Reply:
From what I've read in the docs thus far, I believe we can send the version tag in the body of the HTTP request and parse its value here to use for creating out GitHub tag 👍